src/Symfony/Component/Console/Style/SymfonyStyle.php

```diff
@@ -176,7 +178,7 @@ public function caution($message)
     /**
      * {@inheritdoc}
      */
-    public function table(array $headers, array $rows)
+    public function table(array $headers, array $rows, bool $horizontal = false)
```

Comment:
this breaks inheritance, we need a BC layer

Reply:
I added a new method instead